rules-reviewed/openshift/logging.windup.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<ruleset id="logging"
+    xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This is a ruleset for logging-related topics when migrating and application to cloud environments.
+        </description>
+        <dependencies>
+          <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+          <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="java"/>
+        <sourceTechnology id="java-ee"/>
+        <targetTechnology id="openshift"/>
+        <targetTechnology id="cloud-readiness"/>
+        <tag>logging</tag>
+    </metadata>
+    <rules>
+        <rule id="logging-0000">
+            <when>
+              <or>
+                <filecontent filename="log4j.{extension}" pattern="{appenderpattern}"/>
+                <javaclass references="org.apache.{*}log4j.{*}FileAppender{*}" />
+                <javaclass references="java.util.logging.FileHandler{*}"/>
+                <javaclass references="ch.qos.logback.core.FileAppender"/>
+                <javaclass references="org.pmw.tinylog.writers.FileWriter"/>
+                <javaclass references="org.slf4j.{*}"/>
+                <javaclass references="org.apache.commons.logging.{*}"/>
+              </or>
+            </when>
+            <perform>
+                <iteration>
+                    <hint title="Logging to a file" effort="3" category-id="cloud-mandatory">
+                        <message>
+                                The application logs to a file on a local storage.  
```

Comment:
I would avoid of shortcuts without explanation/definitions.
s/stdout/standard output/